server/handler/renderer.go

```diff
@@ -0,0 +1,44 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/src-d/code-annotation/server/serializer"
+	"github.com/src-d/code-annotation/server/service"
+)
+
+func Write(w http.ResponseWriter, r *http.Request, response *serializer.Response, errs []*serializer.Error) {
+	//TODO: Responses and Redirects should implement Serializable{Serialize(w)}
+	if response.RedirectURL != "" {
+		http.Redirect(w, r, response.RedirectURL, response.Status)
+		return
+	}
+
+	status := http.StatusOK
+
+	if len(errs) > 0 {
+		status = http.StatusInternalServerError
```

Comment:
something is wrong here.
if handler returns error `not found` the status should be `not found` not `internal server error`